ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -1062,11 +1100,30 @@ describe('RuntimeConfigurationPanel', () => {
     await mustClickButton(wrapper, 'Next');
     await mustClickButton(wrapper, 'Update');
 
+    await mustClickButton(wrapper, 'Update');
```

Comment:
why does this test click Update twice?

Reply:
Since its PD it adds an extra page with the persistent disk information and then has the final page which triggers update and delete

Reply:
could you add a comment saying that?